hackathon_site/dashboard/frontend/src/pages/Orders/Orders.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
I think there are some problems while I am trying test your ticket Karandeep. And it might be caused by the conversion from JS to Typescript. 

<img width="735" alt="CleanShot 2022-06-24 at 00 25 27@2x" src="https://user-images.githubusercontent.com/36619969/175462370-442876a5-def4-44ef-8d79-c4f8ce70e7ed.png">

Reply:
Leo, I think you may have changes on your side because it worked fine for me. Maybe to a got stash before pulling?